src/main/java/io/split/android/client/events/SplitEventsManager.java

```diff
@@ -135,6 +157,11 @@ public boolean eventAlreadyTriggered(SplitEvent event) {
         return _executionTimes.get(event) == 0;
     }
 
+    private boolean wasTriggered(SplitInternalEvent event) {
```

Comment:
Do we need `wasTriggered` when we already have `isTriggered` ?
I am not sure why we are using one method in some cases and the other one in others.
If it is possible to remove `wasTriggered`, you can then remove `_triggered` variable and the `ConcurrentSet` class.

Reply:
One function is for internal sdk events and the other is for external evens.There was an inconsistency with the names, but I fixed it by renaming isTriggered.